fingerprint/setup_configurations.md

```diff
@@ -0,0 +1,54 @@
+## Setup and Configuration of Fingerprint
+In order to use the fingerprint functionality you need to have a [Digital Personal](https://www.neurotechnology.com/fingerprint-scanner-digitalpersona-u-are-u-4500.html) 4500 device and its drivers installed.
+### General Setup
+#### Configure Fingerprint app
+In order to configure the fingerprint app there are two things that should be done. 
+ 1. Setting the username and password for the it to access the openmrs database. 
+ 2. UgandaEMR  Settings
+     
+       a ) [Login](../login.md) In UgandaEMR. This will navigate you to the _"Home"_ page
+       
+       b ) Click on the "System Administrator icon ![System Admin icon](../images/system_admin_icon.png). This will navigate you to the _" System Administration"_ page. 
+       
+       c ) Click on the Fingerprint Settings icon. ![Fingerprint Settings icon](../images/fingerprint_settings_icon.png). This will navigate you to the fingerprint settings
+       ![Fingerprint Settings Page](../images/fingerprint_setting_page.png)
+       
+       d ) In order to enable or disable a finger type _"true"_ to enable and _"false"_ to disable. 
+       
+       e ) The Socket IP Address is very important. It should be set to _"http://localhost:8084"_ if its not set, or different, change it to _"http://localhost:8084"_.
+       
+       f ) In order to save the changes, _"click the save"_ button. 
+      
+
+### Client Server Setup
+
+#### Client Machine
+1. Ensure UgandaEMR has been installed successfully: The you can ably login using your session like ART, SMC, OPD and the many more.
+
+2. Server and client computers must be on the same network: Ping the server address from your computer. Ping replies should be successful.
+
+3. [Download fingerprint connector](https://sourceforge.net/projects/ugandaemr/files/Others/fingerprint%20connector.zip/download) on your computer. 
+
+4. Unzip the download to a directory of your choice 
+
+5. In the fingerprint directory. Open the file _"fingerprintrun.bat"_ in any text editor of your choice. [Note pad ++ is preferred](https://notepad-plus-plus.org/).
+
+6. Replace _"localhost"_ with the IP address of the server.
+
+7.  Right click on the  _"fingerprintrun.bat"_ and Create shortcut to Desktop
+
+8. Go to Desktop and rename the Short to "Start Fingerprint". 
+
+9. May Change the icon of the shortcut. There is one provided in the fingerprint directory.
+
+#### Client Machine
+1. Launch mysql client
+2. Run the following commands below
```

Comment:
@slubwama Shouldn't the mysql commands be run on the server, as that is where the access is granted not on the client